performances/performance_test/include/performance_test/ros2/node.hpp

```diff
@@ -41,8 +41,9 @@ friend class System;
 
 public:
 
-  Node(const std::string& name, const std::string& ros2_namespace = "", bool ipc = true)
-    : rclcpp::Node(name, ros2_namespace, rclcpp::NodeOptions().use_intra_process_comms(ipc))
+  Node(const std::string& name, const std::string& ros2_namespace = "",
```

Comment:
This change is breaking one of the unit-tests:
`performance_test/test/test_node.cpp`, you have to change the node constructore there